tutorials/eeg-coding-culture.rst

```diff
@@ -0,0 +1,14 @@
+Coding culture at EEG
+=====================
+
+This talk was given by `Philip Hackstock <https://iiasa.ac.at/staff/philip-hackstock>`_
+on the 17th of January 2022 for the EEG group at the TU Vienna.
+
+Presentation slides (.pptx & .pdf, english): DOI: `10.5281/zenodo.5881657
+<https://doi.org/10.5281/zenodo.5881657>`_
```

Comment:
```suggestion
- DOI: `10.5281/zenodo.5881657 <https://doi.org/10.5281/zenodo.5881657>`_
```
But I guess only personal preference to stay in the same layout like "Example repositories:"